### PR TITLE
Document no prefix for create

### LINKED ISSUE #37: update docs for --no-prefix to specify limitations
instead of `Don't prefix experiment with app_name (supports legacy splits)` should we say something more verbose like `Don't prefix experiment with app_name. Do not use for any new splits, this flag is only meant for modifying existing legacy splits`?

### PR DESCRIPTION
**Pull request by jmileham**
/domain @samandmoore for testtrack v1.0 - we ready?
/domain @rnackman @joejansen for the doc fixes - does this work? I wanted to keep the parameter documentation tight, so i put `existing` in there to prompt doubt, and then put the full details in the `--help` documentation for both experiment and feature_flag creation commands.

fixes #37

### CODE REVIEW

**Comment by samandmoore** [comment]
<<domainlgtm

Works for me. I think 1.0 makes sense.

**Comment by jmileham** [comment]
@aburgel notes that we don't have compatibility with the latest alpha of the rails client because there's no v2/split_registry endpoint in the fake. So I'm going to walk back calling this 1.0. We should have those in place by the end of the week and be good to go then.

File: cmds/create_experiment.go [hunk 1]
```diff
@@ -23,13 +23,17 @@ weightings of 50% each.
 Weights are specified as a string and must sum to 100:
 
 --weights "variant_1: 25, variant_2: 25, variant_3: 50"
+
+Do not use --no-prefix to create a new split. It can be used to revive a
+destroyed split if it was destroyed by mistake, but the migration will fail if
+you attempt to create a new split without a prefix.
```
Comment (rnackman) [inline cmds/create_experiment.go]: 👍

**Comment by rnackman** [comment]
<< domain lgtm -- one wording suggestion that you can take or leave for the parameter doc!

File: cmds/create_experiment.go [hunk 2]
```diff
@@ -23,13 +23,17 @@ weightings of 50% each.
...
 `
 
 var createExperimentWeights string
 
 func init() {
 	createExperimentCmd.Flags().StringVar(&createExperimentWeights, "weights", "control: 50, treatment: 50", "Variant weights to use")
-	createExperimentCmd.Flags().BoolVar(&noPrefix, "no-prefix", false, "Don't prefix experiment with app_name (supports legacy splits)")
+	createExperimentCmd.Flags().BoolVar(&noPrefix, "no-prefix", false, "Don't prefix experiment with app_name (supports existing legacy splits)")
```
[the remark this answers is not in the document]
Comment (jmileham) [reply cmds/create_experiment.go]: Thanks for the push to be more descriptive. You could be reviving or amending the weights of a non-destroyed legacy split, too, so I don't want to be over prescriptive of the use case. I'm hoping `existing` makes people appropriately worried about using the feature to create a new split.

**Comment by jmileham** [comment]
Y'know what, since @aburgel's planning on adding v2 endpoint support this week, I'm not even going to bump version or release this standalone. No need for the churn. We'll sweep this in with 1.0 when that is ready to go. Going to merge this. Thanks all!

**Pull request merged**